### DISCUSSION TITLE
Add support of note objects

### DISCUSSION THREAD

**Discussion by aacebedo** [structurizr/java#273] [Q&A]
### Description

HI

I am trying structurizr and was wondering if it would be possible to add the support of a note object like the UML one. 
Its difference with a custom element would be:
- Have the top right corner folded like a post-it
- No title, only description
- Relationship without arrows

### Priority

I have no budget and there's no rush, please add this feature for free

**Comment by simonbrowndotje** [comment]
I'm not a fan of UML's "note" feature, and the resulting notes are often mistaken for UML elements, so this isn't something I'm planning to add. I'd recommend using the interactive tooltips for supplementary information/documentation instead.

**Reply by aacebedo** [reply]
Thanks for the quick answer !

The thing is that I would like to use extracted images in pdf documents so they are not interactive. In this case, there is no simple way to add comments or contextual doc to the diagram without an actual note element.

**Reply by aacebedo** [reply]
If I add the feature myself. Would it be accepted?

**Reply by jagoodhand** [reply]
@simonbrowndotje - is there a way to add additional detail to a tooltip without using perspectives (which don't show unless the perspective is active) or properties (which don't seem fitted to content beyond a few words)?

The model in my head is that the current 'description' property is more like a 'short description'. It would be nice to be able to add a few more sentences of detail to an item in the tooltip without having a full markdown document.

This would mean you then had something like:
- description (current 'description' property)
- additional detail (missing)
- specific detail (perspectives)
- documentation (markdown document)
